ConstructedScripts/shavian_two_layer.yaml

```diff
@@ -0,0 +1,32 @@
+name: "Shavian 2-layer"
+rows:
```

Comment:
You should add autoShift: false somewhere on the top level if you're using shift as a two-layer system instead of actual capitalization. Otherwise auto-capitalization on some text fields will automatically enter you into shifted layer and it can be annoying